src/views/Projects/index.jsx

```diff
@@ -20,11 +21,32 @@ import ProjectCard from '../../components/ProjectCard';
 export default class Projects extends Component {
   state = {
     projects,
+    search: '',
+  };
+  handleTextInputChange = event => {
+    this.setState({ search: event.target.value });
   };
 
   render() {
     const { classes } = this.props;
-    const { projects } = this.state;
+    const { projects, search } = this.state;
+    const selectedProjects = Object.keys(projects)
+      .filter(fileName => {
+        const found =
+          fileName.toLowerCase().indexOf(search.toLowerCase()) !== -1 ||
```

Comment:
A user, in theory, has no knowledge of filenames so we don't need to filter on them. Filtering on project names is good enough.